impl/src/main/java/org/glassfish/soteria/identitystores/LdapIdentityStore.java

```diff
@@ -412,6 +445,24 @@ private static String ensureFilterHasParens(String filter) {
         }
     }
 
+    private static void processNamingException(NamingException e) throws Exception {
+      String exString = e.toString();
+      if (exString !=null && exString.indexOf("LDAP: error code 48") >= 0) {
+        AuthenticationException ae = new AuthenticationException(e.getMessage());
+        ae.initCause(e);
+        throw ae;
+      }
+      // TBD add other cases which also belong to config error.
+      else if (exString !=null && exString.indexOf("LDAP: error code 32") >= 0) {
```

Comment:
Same comment as for error code 48 -- this is not really necessary given the richness of the `NamingException` hierarchy, and it's also brittle -- the message might be different if localized for a different language, or as implemented by a different JDK (the IBM JDK, or the HP JDK).